File: AspNetMvcUnleashed/0672329980_source/Chapter10/VB/MvcApplication1/MvcApplication1/Controllers/ProductController.vb
Public Class ProductController
    Inherits Controller

    <FeaturedProduct()> _
    Public Function Index() As ActionResult
        Return View()
    End Function

    <FeaturedProduct()> _
    Public Function Details() As ActionResult
        Return View()
    End Function


    Public Function About() As ActionResult
        Return View()
    End Function

End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter10/CS/MvcApplication1/MvcApplication1/Controllers/ProductController.cs
using System.Web.Mvc;
using MvcApplication1.CustomActionFilters;

namespace MvcApplication1.Controllers
{
    public class ProductController : Controller
    {

        [FeaturedProduct]
        public ActionResult Index()
        {
            return View();
        }

        [FeaturedProduct]
        public ActionResult Details()
        {
            return View();
        }

    
        public ActionResult About()
        {
            return View();
        }

    }
}
